src/muse/mca.py

```diff
@@ -128,9 +128,9 @@ def __init__(
         carbon_price: Sequence | None = None,
         carbon_commodities: Sequence[str] | None = None,
         debug: bool = False,
-        control_undershoot: bool = True,
-        control_overshoot: bool = True,
-        carbon_method: str = "fitting",
+        control_undershoot: bool = False,
+        control_overshoot: bool = False,
+        carbon_method: str = "bisection",
```

Comment:
Probably not many people is using it, but as you are changing the defaults, make sure that this change is clearly explained in the next release notes.